### DISCUSSION TITLE
[@types/react-datetime-picker] minDate missing in typings for DateTimePickerProps

### DISCUSSION THREAD

**Discussion by Coddielam** [DefinitelyTyped/DefinitelyTyped#57568] [Issues with a @types package]
Hi,
I am using react-datetime-picker@3.4.3 and have installed @types/react-datetime-picker@3.4.0 (latest as of now). In the react-datetime-picker documentations -- in both 3.4.0 and 3.4.3 -- there's the minDate prop but it was not included in the typings defined in DateTimePickerProps
![Screenshot 2021-12-06 at 12 26 36 PM](https://user-images.githubusercontent.com/43496495/144787137-eb51d8d8-a1d0-46e6-8824-24255adea524.png)
.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Done #57584
